### DISCUSSION TITLE
CD icon for CD and a CD on a BLANK PAPER as CD IMAGE

### DISCUSSION THREAD

**Discussion by nestoris** [nestoris/Win98SE#5] [Announcements]
I want to use this: ![cdimage](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/devices/64/media-optical.png) icon for inserted Compact Disk or DVD and another icon (something like this of [Grassmunk](https://github.com/grassmunk/): ![cdimage](https://raw.githubusercontent.com/grassmunk/Chicago95/master/Icons/Chicago95/mimes/48/application-x-cd-image.png) for IMG, ISO, NRG, CUE, etc images.
Be prepared! :-) Or write here what do you mind about it.

**Comment by Craftplacer** [comment]
You could consider using a different media format for `.IMG` files as they're mostly partition images (e.g. disks and floppies, but not limited to CDs).

**Reply by nestoris** [reply]
ok

**Reply by nestoris** [reply]
`application/vnd.squashfs` is the same as `.img`?

**Reply by Craftplacer** [reply]
Well it's a filesystem, so I would say it serves the same purpose as IMG

**Comment by nestoris** [comment]
It's done. For now there are: ![cdimage](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/mimes/64/application-x-cd-image.png) for cd-images and ![fsimage](https://raw.githubusercontent.com/nestoris/Win98SE/main/SE98/mimes/64/application-x-raw-disk-image.png) for file systems images.
